src/aics-image-viewer/components/StyleProvider/index.tsx

```diff
@@ -155,18 +166,23 @@ const CssProvider = styled.div<{ $theme: AppTheme }>`
 
       --color-button-link-text: ${$theme.colors.button.link.text};
 
-      --color-button-default-bg: transparent;
-      --color-button-default-text: ${$theme.colors.button.secondary.text};
-      --color-button-default-outline: ${$theme.colors.button.secondary.outline};
-      --color-button-default-hover-outline: ${$theme.colors.button.secondary.hoverOutline};
-      --color-button-default-hover-text: ${$theme.colors.button.secondary.hoverText};
-      --color-button-default-active-outline: ${$theme.colors.button.secondary.activeOutline};
-      --color-button-default-active-text: ${$theme.colors.button.secondary.hoverText};
```

Comment:
`default` -> `tertiary`

Reply:
Why are there so much fewer variables for `secondary` than `tertiary`? Do secondary buttons just deviate less from the global button styling below?

Reply:
Secondary derives almost entirely from primary, yes! They behave similarly when hovered and active.

![image](https://github.com/allen-cell-animated/website-3d-cell-viewer/assets/30200665/ce976e10-d619-4b72-8b47-f9332fbd871f)

Reply:
Oops, thought I'd responded to this but evidently not.
Secondary buttons derive mostly from primary buttons, except when at rest or disabled.

![image](https://github.com/allen-cell-animated/website-3d-cell-viewer/assets/30200665/ea3baa05-eb26-47f3-9bd5-ff5395b69965)